### PR TITLE
replace councils page with temporary page

### PR DESCRIPTION
**Pull request by wrightmartin**
Temporarily removes the councils page as the information was out of date.

Replaces it with a holding page

New page is on the way

### CODE REVIEW

**Comment by wrightmartin** [comment]
Looks like this

screen shot 2016-08-02 at 12 01 52

**Comment by wrightmartin** [comment]
@davea in @dracos' absence would you mind look at this for us?

File: templates/web/fixmystreet.com/about/council.html [hunk 1]
```diff
@@ -12,259 +12,10 @@
...
-</div>
-<div class="councils-section call-to-action">
-    <div class="fixed-container">
-            <h2>Book a demo</h2>
-            <p>Want to know more? <a href="mailto:enquiries@mysociety.org">Get in touch now</a> &ndash; we’re always up for a chat about council needs.</p>
-            <p>Or, to really understand how FixMyStreet for Councils could work for you, book a demo. We’ll show you exactly how it would look and behave on your own website. Just drop us a line.</p>
-            <a href="mailto:enquiries@mysociety.org" class="button">Get in touch</a>
+        <q class="hero-quote">To improve is to change; to be perfect is to change often.</q>
+        <cite class="hero-cite">Winston Churchill</cite>
+        <p class="hero-whatsup">We're currently working on this page to reflect our new and improved offering. In the meantime, we've got all of our product information available on gov.uk's Digital Marketplace</p>
```
Comment (davea) [inline templates/web/fixmystreet.com/about/council.html]: Missing full stop (sorry).

**Comment by davea** [comment]
On narrow screens the marketplace button text could do with being centred, and the 'Reporting a problem?' banner at the top gets a bit lost in all the yellow:
screen shot 2016-08-02 at 15 57 33

File: templates/web/fixmystreet.com/about/council.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     <div class="fixed-container">
         <q class="hero-quote">To improve is to change; to be perfect is to change often.</q>
         <cite class="hero-cite">Winston Churchill</cite>
-        <p class="hero-whatsup">We're currently working on this page to reflect our new and improved offering. In the meantime, all of our product information is available on GOV.UK's Digital Marketplace.</p>
+        <p class="hero-whatsup">We're currently working on this page to reflect our new and improved offering. In the meantime, all of our product information is available on the GOV.UK Digital Marketplace.</p>
```
Comment (BenJam) [inline templates/web/fixmystreet.com/about/council.html]: 'the government's Digital Marketplace' plz.

**Comment by dracos** [comment]
This has been merged as d817711

**Pull request closed without merging**